### DISCUSSION TITLE
Overview of ALL flaws

### DISCUSSION THREAD

**Discussion by peterbe** [mdn/mdn-community#585] [Content]
When you run `yarn start` you can navigate to a specific page and Yari renders it on the fly and "logs" all flaws. These flaws are listed in the toolbar (clicking "Show flaws"). 

But to see a complete listing of ALL flaws, you have to *build all documents*. 

Thanks to https://github.com/mdn/content/pull/6349 you now can...

### To use

You'll need to start 2 terminals. 

1. In the first one, `yarn install && yarn start` as usual and then go to http://localhost:5000/en-US/_flaws (or http://localhost:5000/ and scroll to the "Flaws Dashboard" link)
2. Type `yarn build` and wait a little. Now go *back* to  http://localhost:5000/en-US/_flaws in your browser and you should see it listing documents with >0 flaws. 

### Tips

The `yarn build` command is slow. Suppose you're only interested in, say, JavaScript Global Objects. You can then use:
```
BUILD_FOLDERSEARCH=web/javascript/reference/global yarn build
```

Every time you *return* to your browser at http://localhost:5000/en-US/_flaws it will trigger a reload. You'll see a small "Reloading..." in the upper left-hand corner whenever you focus the window. No need to refresh the page manually. 

[Soon](https://github.com/mdn/yari/pull/4258), you'll be able to to use `yarn build --no-html` which means it doesn't bother generating the full `index.html` file for each document.

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
